src/main/proto/netflix/titus/titus_task_relocation_api.proto

```diff
@@ -17,62 +17,101 @@ option go_package = "titus";
 // ----------------------------------------------------------------------------
 // Task relocation core data structures
 
-/// A single relocation plan.
-message RelocationPlan {
-    /// (Required) A list of task ids to terminate.
-    repeated string candidateTaskIds = 1;
+message TaskRelocationPlan {
+    /// (Required) Id of a task to relocate.
+    string taskId = 1;
+
+    /// (Required) Relocation reason code. Currently defined reason codes are:
+    //  * 'migrationFromDeprecatedAgent'
```

Comment:
Remove reason codes until we figure out the different reasons? These names are based on older concepts

Reply:
Ok, removed